Description: Add cpu feature xfd.
Author: Tim Wiederhake <twiederh@redhat.com>
Origin: upstream, https://gitlab.com/libvirt/libvirt/-/commit/29b0b9bb0eb966f9c49b78b97101c6623306a39a
Bug-Ubuntu: https://launchpad.net/bugs/2028057
Forwarded: not-needed
Last-Update: 2023-11-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/cpu_map/x86_features.xml
+++ b/src/cpu_map/x86_features.xml
@@ -430,6 +430,9 @@
   <feature name='xsaves' migratable='no'>
     <cpuid eax_in='0x0d' ecx_in='0x01' eax='0x00000008'/>
   </feature>
+  <feature name='xfd'>
+    <cpuid eax_in='0x0d' ecx_in='0x01' eax='0x00000010'/>
+  </feature>
 
   <!-- cpuid level 0x0000000f:1 (edx) -->
   <feature name='mbm_total'>
